src/Uno.UI/UI/Xaml/Media/Imaging/SvgImageSource.cs

```diff
@@ -56,7 +56,12 @@ private void OnUriSourceChanged(DependencyPropertyChangedEventArgs e)
 		{
 			UnloadImageData();
 		}
-		InitFromUri(e.NewValue as Uri);
+
+		if (e.NewValue is Uri uri)
+		{
+			InitFromUri(uri);
+		}
```

Comment:
If we are handling `null` in `InitFromUri`, then we don't need the if condition here

Reply:
We still need the cast, that's fine if we check for null as well.